deploy/ironic_bmo_configmap.yaml

```diff
@@ -3,11 +3,11 @@ apiVersion: v1
 metadata:
   name: ironic-bmo-configmap
 data:
-  http_port: "6180"
-  interface: "eth2"
-  dhcp_range: "172.22.0.10,172.22.0.100"
-  deploy_kernel_url: "http://172.22.0.2:6180/images/ironic-python-agent.kernel"
-  deploy_ramdisk_url: "http://172.22.0.2:6180/images/ironic-python-agent.initramfs"
-  ironic_endpoint: "http://172.22.0.2:6385/v1/"
-  ironic_inspector_endpoint: "http://172.22.0.2:5050/v1/"
-  cacheurl: "http://172.22.0.1/images"
+  HTTP_PORT: "6180"
+  INTERFACE: "eth2"
+  DHCP_RANGE: "172.22.0.10,172.22.0.100"
+  DEPLOY_KERNEL_URL: "http://172.22.0.2:6180/images/ironic-python-agent.kernel"
+  DEPLOY_RAMDISK_URL: "http://172.22.0.2:6180/images/ironic-python-agent.initramfs"
+  IRONIC_ENDPOINT: "http://172.22.0.2:6385/v1/"
+  IRONIC_INSPECTOR_ENDPOINT: "http://172.22.0.2:5050/v1/"
+  CACHEURL: "http://172.22.0.1/images"
```

Comment:
Cc @sadasu @imain do you see any issues with switching the interface in the MAO to define the map of environment variables directly, then use envFrom as below?

The idea is to avoid wiring every variable in explicitly to each container, instead we can just pass the entire configmap into each.